src/triage/component/postmodeling/fairness/aequitas_utils.py

```diff
@@ -0,0 +1,25 @@
+import pandas as pd
+
+def get_aequitas_results(engine, parameter, schema="test_results", table="aequitas", model_id=None,  subset_hash="", tie_breaker="worst"):
+    ''' This function returns the current contents of the aequitas table.
+
+        Args:
+            - engine: SQLAlchemy engine conected to database
+            - parameter: A string that indicates any parameters for the metric (ex. `100_abs` indicates top-100 entities)
+            - schema: Databse schema to find table within
+            - table: Databse table to select data from
+            - model_id: A model_id, to query only for results of that model
+            - subset_hash: Identifies the subset for the evaluation
+            - tie_breaker: Indicates how ties are broken
+
+        Returns: A DataFrame, corresponding to schema.table
+    '''
+
+    query = f"SELECT * FROM {schema}.{table} WHERE parameter = '{parameter}'"
+    if model_id:
+        query += f" AND model_id = {model_id}"
+    if subset_hash:
+        query += f" AND subset_hash = '{subset_hash}'"
+    if tie_breaker:
+        query += f" AND tie_breaker = '{tie_breaker}'"
```

Comment:
Probably wouldn't treat these as optional parameters since you'll need them to avoid duplication in the result set.

Also note that `if subset_hash` will be treated as `if False` when `subset_hash=""` (the default) even though we want to be able to select on that value (though here I would remove the if statement entirely anyway).